autoload/bracey.vim

```diff
@@ -32,7 +32,7 @@ endfunction
 function! bracey#startBrowser(url)
 	if g:bracey_browser_command == 0
 		if has("unix")
-			if system("uname -s") == "Darwin"
+			if system("uname -s") == "Darwin\n"
```

Comment:
I think it would be better to strip the leading/trailing white space from the result of system. from the result of system.

Reply:
I've "tried" fixing it maybe that works? 

~= instead of ==